app/controllers/follow_requests_controller.rb

```diff
@@ -1,9 +1,15 @@
 class FollowRequestsController < ApplicationController
-  before_action :set_follow_request, only: %i[ show edit update destroy ]
+  before_action :set_follow_request, only: %i[show edit update destroy]
+  before_action :authorize_resource, except: [:index, :new, :create]
```

Comment:
Only error I found was when I loaded the web page and i tried to follow someone I was not following, I would get the "Pundit::AuthorizationNotPerformedError at /follow_requests
FollowRequestsController" Error. Potentially could be because you are creating a like but you are skipping the authorization on :create. It could be this line but I could be wrong.

Reply:
Good catch!